cypress/support/override-commands.ts

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
Just a place holder with a doc description and link to cypress documentation on overrides.